### PR TITLE
Created response table markdown doc for NIM

### PR DESCRIPTION
**Pull request by EthanLettuce**
I wrote a little document for operators to easily lookup the NIM error codes and responses.

### CODE REVIEW

File: doc/NIM_Response_Table.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Northern Images (NIM) Response Table
+The following two tables contains all of the possible responses and error codes from the NIM payload.
+git
```
Comment (Taylorr82) [inline doc/NIM_Response_Table.md:3]: git

File: doc/NIM_Response_Table.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Northern Images (NIM) Response Table
+The following two tables contains all of the possible responses and error codes from the NIM payload.
```
Comment (Taylorr82) [inline doc/NIM_Response_Table.md:2]: Should specify that error codes are received from the satellite as negative numbers

**Review by Taylorr82 — CHANGES_REQUESTED** [review]

File: doc/NIM_Response_Table.md (hunk 2 above)
Comment (EthanLettuce) [reply doc/NIM_Response_Table.md:2]: Okay fair enough

File: doc/NIM_Response_Table.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+git
+## Error codes given by the OBC
+| Decimal | Hex | Error |
+|:-------:|:---:|:-----|
+| 0 | 0x00 | OK |
+| 1 | 0x01 | Fail |
+| 2 | 0x02 | Hanlder Busy |
```
Comment (Taylorr82) [inline doc/NIM_Response_Table.md:9]: Misspelled handler

**Review by Taylorr82 — APPROVED** [review]

**Pull request merged**